plop-templates/Component/Component.tsx.hbs

```diff
@@ -30,10 +30,6 @@ function {{pascalCase name}}({ children, align }: Props) {
     </div>
   );
 }
-
-{{pascalCase name}}.defaultProps = {
-  children: "Hello world =^_^=",
-  align: "left",
-};
```

Comment:
move these into the function declaration above

Reply:
oops